### PR TITLE
Api

### PR DESCRIPTION
**Pull request by kene-creator**
- Created a new Rails API app called 'hello-rails-back-end'.
- Initialized my project with Git.
- Made sure that your project has a Postgres database set up.
- Created a table for storing your messages and created 5 different greetings.
- Created an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).

### CODE REVIEW

**Review by fobadara — COMMENTED** [review]
Hi @kene-creator, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a future features section, an acknowledgements section and has the microverse logo in the README.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/controllers/api/v1/messages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module Api
+  module V1
```
Comment (Davidosky007) [inline app/controllers/api/v1/messages_controller.rb:2]: ## Kindly display random greetings according to the [project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md#:~:text=Display%20the%20random%20greeting%20in%20your%20Greeting%20component.) as currently, you are rendering all the greetings at once, bellow  👇 are screenshots of my suggestions on fixing these issues  
image

image

image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @kene-creator 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights

* No linter errors 👍 
* Good looking readme file 👍 
* Proper use of Gitflow 👍 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/api/v1/messages_controller.rb (hunk 1 above)
Comment (kene-creator) [reply app/controllers/api/v1/messages_controller.rb:2]: Thank you for your reply🤩😊

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @kene-creator,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**